server/methods/userClasses.js

```diff
@@ -0,0 +1,44 @@
+Meteor.methods({
+  updateUserClass: function (modifier, documentId) {
+    Class.update({ _id: documentId }, modifier);
+    var newUserClass = Class.findOne({ _id: documentId });
+    UserClass.update({ classId: documentId }, {
+      $set: {
+        title: newUserClass.title,
+        shortTitle: newUserClass.shortTitle
+      }
+    });
+    User.update({ 'classes.classId': documentId }, {
+      $set: {
+        'classes.$.title': newUserClass.title,
+        'classes.$.shortTitle': newUserClass.shortTitle
+      }
+    }, { multi: true });
+  },
+  createUserClass: function (doc) {
+    Schemas.class.clean(doc);
+    Class.insert(doc);
+    UserClass.insert({
+      title: doc.title,
+      shortTitle: doc.shortTitle,
+      classId: Class.findOne({ title: doc.title }, { _id: 1 })._id
+    });
+  },
+  editUserClasses: function (doc) {
```

Comment:
The naming is kinda ambiguous, see my previous comments on the schema? Also consider adding comments.